README.md

```diff
@@ -1,10 +1,10 @@
 # Welcome to the Windows Bridge for iOS Project
-The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices by re-using your Objective-C code and iOS APIs alongside Windows 10 features like Cortana and Live Tiles.
+The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices by re-using your Objective-C code and iOS APIs alongside Windows 10 features like Cortana and Windows Notifications.
 
 ## Download and Installation
-To use the bridge, you'll need:
-- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest) (click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice when complete).
-- **Windows 10**, build 10586 or higher (validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system)).
+To use the bridge you'll need:
+- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest). Click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice when complete.
```

Comment:
Suggestion: point the user to a specific landing directory as an example so you can use it when showing your command sample.

...local directory of your choice (example: **c:\winobjc**) when complete.